### DISCUSSION TITLE
Generate Training job package without included frames

### DISCUSSION THREAD

**Discussion by ghost** [talmolab/sleap#947] [Help!] [answered]
Hi

I want to run SLEAP on Colab and wonder if it is possible to generate the Training job package in GUI (or by using any other way) without including the labeled frames and get frames from the video existing in Google drive (I want just include the labels, not frames). I have my video in Google drive and I want to reduce the size of the generated Training job package.
When I do inference in colab by using the existing video file in Google drive, the size of the generated .slp file containing the predicted labels, and the size of .slp file is very low (like 100 KB). However, the generated Training job package by GUI including the labeled frame is in the order of 30-40 MB.

Thanks for your help

**Comment by roomrys** [comment, marked as answer]
Hi @aarahvar2,

In the training pipeline, instead of selecting to "Export training job package...", you can instead select to "Save configuration files...". This will create a directory similar to the one for the training package, except no labels package (`.pkg.slp`) is included and you should upload the labels project (`.slp`) to Colab.

Let us know how that works!

Thanks,
Liezl

**Reply by ghost** [reply]
Thanks Liezl. Great, it worked.
